HARP-2023-Summer/Mapping/Functions/fn_tablegen.R

```diff
@@ -52,7 +52,11 @@ fn_tablegen <- function(featr_type, table, columns, alignment, origin_type, metr
     ft <- align(ft, align = alignment, part = "all")
     
     #highlight when precent diff is below highlight limit(defined in config)
+    
+    if (data_set == 'rseg_no_geom')  { #only do this for rseg maps not facil maps 
```

Comment:
Hey @EllaF21 -- great work getting the changes made in all of these sections, it looks like it will work nicely. My only concern is this particular one where we are being very very bespoke, as in, we are targetting a single column for highlighting, and also, only highlighting for a single data_set -- this needs to be way more robust.  If highlighting is a thing we want, we need to be able to tell it to enable highlighting (TRUE/FALSE), what column to use for hightlighting % difference is definitely *not* the only column that might guide highlighting) and what is(are) the threshold(s), i.e. a single highlight or multiple thresholds (red, orange, yelow etc) -- see `om_cu_table()` for an example https://github.com/HARPgroup/hydro-tools/blob/master/R/om_cu_table.R